code/src/components/Buttons.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export const Buttons = ({ section, setSection }) => {
+  const nextSection = (event) => {
+    event.preventDefault();
+    setSection(section + 1);
+  };
+
+  const previousSection = (event) => {
+    event.preventDefault();
+    setSection(section - 1);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setSection(5);
+  };
+  return (
+    <div className="btn-container">
```

Comment:
´This is the section where I said it's different from Daniel's code. He did all the conditional on App.js but you did it on buttons. May I know why you decided so?